src/components/organize/journeys/JourneyInstanceSummary.tsx

```diff
@@ -64,51 +64,62 @@ const JourneyInstanceSummary = ({
     );
   };
 
-  const submitChange = () => {
+  const cancelEditingSummary = () => {
     setEditingSummary(false);
-    saveEditedSummary(summary);
+    setSummary(journeyInstance.summary);
   };
 
   return (
     <ZetkinSection
       action={
-        <Button
-          color="primary"
-          data-testid="JourneyInstanceSummary-saveEditButton"
-          onClick={
-            editingSummary ? submitChange : () => setEditingSummary(true)
-          }
-          startIcon={editingSummary ? <Save /> : <Edit />}
-          style={{ textTransform: 'uppercase' }}
-        >
-          <Msg
-            id={
-              editingSummary
-                ? 'pages.organizeJourneyInstance.saveButton'
-                : 'pages.organizeJourneyInstance.editButton'
-            }
-          />
-        </Button>
+        editingSummary ? null : (
+          <Button
+            color="primary"
+            data-testid="JourneyInstanceSummary-editButton"
+            onClick={() => setEditingSummary(true)}
+            startIcon={<Edit />}
+            style={{ textTransform: 'uppercase' }}
+          >
+            <Msg id={'pages.organizeJourneyInstance.editButton'} />
+          </Button>
+        )
       }
       title={intl.formatMessage({
         id: 'pages.organizeJourneyInstance.sections.summary',
       })}
     >
       {editingSummary ? (
-        <ZetkinAutoTextArea
-          ref={editingRef}
-          data-testid="JourneyInstanceSummary-textArea"
-          onChange={(value) => setSummary(value)}
-          placeholder={summaryPlaceholder}
-          value={summary}
-        />
+        // Editing
+        <form
+          onSubmit={async (e) => {
+            e.stopPropagation();
+            e.preventDefault();
+            const patchedInstance = await saveEditedSummary();
+            setEditingSummary(false);
+            setSummary(patchedInstance.summary);
```

Comment:
I did it this way so that the internal summary state is updated to the state from the backend.